### ISSUE TITLE
Add instant replay controls

### ISSUE THREAD

**Issue by timotheeg** [timotheeg/nestrischamps#126]
Idea from TomelloSoulMan

> I came up with a crazy idea, that I don't know if it's very hard to develop or even feasible at all... I was thinking how cool would be for restreamers to show highlights of a particular game after it has ended, as it's done in sport broadcasts. I learnt recently you can do some kind of instant replays in OBS, just after the thing you wanna replay has happened, by using the replay buffer, but it's only useful in the short therm, and that would interfere with the game restream. Instead of that, I tought that if the restreamer had some kind of control in the NTC game room that allows setting up "bookmarks" in the current game, that could be used to show them all after the game has finished, with a special layout or something... Since NTC keeps track of the piece sequences (as seen in the recently added controls in replays for Stack Rabbit and those things) that could be used to show a particular replay from T-5 pieces (being T the time or piece sequence of the bookmark), even showing them in slow motion... I think this could be an easy replay system to be managed by one single person, since the only thing the restreamer has to do when they see some cool thing happening (end of a great dig, cool burn, great piece spin) is clicking "bookmark" and switching to another OBS scene when the game ends to show the replays. I know I'm not even a restreamer 🤣  but I guess this could be a great feature, mostly because it's never seen before, at least in regular restreams 
> 
> Obviously the thing you would lose in replays is player cams, but you can't get everything I guess

[Discord message link](https://discord.com/channels/817528744565932043/817528917590016020/989876360283562024)

**Comment by alex-ong** [comment]
Yep, this is definitely cutting edge. Using AI too to figure out highlights would be a +1 on top.
